kudos-services/src/main/java/org/exoplatform/kudos/entity/KudosEntity.java

```diff
@@ -45,7 +45,10 @@
         + " ORDER BY k.createdDate DESC"),
     @NamedQuery(name = "Kudos.countKudosByPeriodAndReceiver", query = "select count(k) from Kudos k"
         + " WHERE k.receiverId = :receiverId"
-        + " AND k.isReceiverUser = :isReceiverUser" + " AND k.createdDate > :startDate" + " AND k.createdDate < :endDate") })
+        + " AND k.isReceiverUser = :isReceiverUser" + " AND k.createdDate > :startDate" + " AND k.createdDate < :endDate"),
+    @NamedQuery(name = "Kudos.countKudosByPeriodAndReceivers", query = "select k.receiverId,count(k) from Kudos k"
+        + " WHERE k.receiverId IN :receiversId" + " AND k.createdDate > :startDate"
```

Comment:
start date must be inclusive and end date exclusive IMO
```suggestion
        + " WHERE k.receiverId IN :receiversId" + " AND k.createdDate >= :startDate"
```